virtwho/migrate/migrateconfiguration.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+"""
+For moving variables from system environment to general config file
+
+Copyright (C) 2020 William Poteat <wpoteat@redhat.com>
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+
+import os
+
+SYSCONFIG_FILENAME = '/etc/sysconfig/virt-who'
+GENERAL_CONFIG_FILENAME = '/etc/virt-who.conf'
+
+def main():
+    migrate_env_to_config()
+
+def migrate_env_to_config(sysconfig_filename=None, general_config_filename=None):
+    if not sysconfig_filename:
+        sysconfig_filename = SYSCONFIG_FILENAME
+    if not general_config_filename:
+        general_config_filename = GENERAL_CONFIG_FILENAME
+
+    interval = None
+    debug = None
+    one_shot = None
+
+    # read know env variables
+    if os.environ.get('VIRTWHO_INTERVAL'):
+        interval = os.environment['VIRTWHO_INTERVAL']
+    if os.environ.get('VIRTWHO_DEBUG'):
+        debug = os.environ['VIRTWHO_DEBUG']
+    if os.environ.get('VIRTWHO_ONE_SHOT'):
+        one_shot = os.environ['VIRTWHO_ONE_SHOT']
+
+    # read the values in the existing /etc/sysconfig/virt-who file
+    # these would override the env variables
+    env_vars = {}
+    with open(sysconfig_filename) as myfile:
+        for line in myfile:
+            if not line.startswith('#'):
+                name, var = line.partition("=")[::2]
+                env_vars[name.strip()] = var
+
+    if env_vars.get('VIRTWHO_INTERVAL'):
+        interval = env_vars.pop('VIRTWHO_INTERVAL')
```

Comment:
You use `pop` here, because you want to save `interval` in `[global]` section and the other env. vars. will be saved in `[system_environment]` section, right? If so, then please add some explanation comment.